saleor/graphql/tax/mutations/tax_configuration_update.py

```diff
@@ -119,12 +139,14 @@ def update_countries_configuration(cls, instance, countries_configuration):
             data = input_data_by_country[obj.country]
             obj.charge_taxes = data["charge_taxes"]
             obj.display_gross_prices = data["display_gross_prices"]
+            obj.tax_calculation_strategy = data["tax_calculation_strategy"]
```

Comment:
As the `tax_calculation_strategy` is not required it might raise an error when there will be not `taxCalculationStrategy` in the input, am I right? IMO, we should use `get` here.

Reply:
So it seems like there is also a lack of test for this situation.

Reply:
Good catch 👍

Reply:
Fixed